ramp-database/rampdb/model/fold.py

```diff
@@ -17,9 +17,26 @@
 
 
 class CVFold(Model):
-    """Storing train and test folds, more precisely: train and test indices.
+    """CVFold table.
 
-    Created when the ramp event is set up.
+    Storing train and test folds, more precisely: train and test indices.
```

Comment:
This description is not fully clear to me.

So if I understand correctly: this table stores the train and test indices for a *single* fold ? And then each event has a bunch or records in this table, corresponding to the number of folds in the CV of that problem?